connector/src/thegraph/connector.ts

```diff
@@ -20,31 +20,33 @@ export function subgraphUrlFromChainId(chainId: number) {
   return null
 }
 
+type ConvictionVotingConnectorTheGraphConfig = {
+  pollInterval?: number
+  subgraphUrl?: string
+  verbose?: boolean
+}
+
+
 export default class ConvictionVotingConnectorTheGraph
   implements IConvictionVotingConnector {
   #gql: GraphQLWrapper
 
-  constructor(subgraphUrl: string, verbose: boolean = false) {
-    if (!subgraphUrl) {
+  constructor(config: ConvictionVotingConnectorTheGraphConfig) {
+    if (!config.subgraphUrl) {
       throw new Error(
         'ConvictionVotingConnector requires subgraphUrl to be passed.'
       )
     }
-    this.#gql = new GraphQLWrapper(subgraphUrl, verbose)
+    this.#gql = new GraphQLWrapper(config.subgraphUrl, {
+      pollInterval: config.pollInterval,
+      verbose: config.verbose,
+    })
   }
 
   async disconnect() {
     this.#gql.close()
   }
 
-  async config(id: string): Promise<Config> {
```

Comment:
On the same lines as the above comment. Wonder if another solution would be to keep this method and advise to use the connector with `onConfig` for reliability.

Reply:
ah the reason why i did that is that quering the config only once is kind of bad because the configs might change in the next second after the query and you will have your app in a wrong state